lalrpop/src/tok/mod.rs

```diff
@@ -83,6 +83,7 @@ pub enum Tok<'input> {
     Star,
     TildeTilde,
     Underscore,
+    ErrorT
```

Comment:
Got a conflict with the `Error` struct defined in this module so went with the ugly `ErrorT` name.